packages/snap-controller/src/Abstract/AbstractController.test.ts

```diff
@@ -535,11 +535,22 @@ describe('Search Controller', () => {
 			filename: 'https://snapui.searchspring.io/test.js',
 		});
 
+		const addedDetails = {
+			stuff: 'and things',
+		};
+
 		const trackerTrackError = jest.spyOn(controller.tracker.track, 'error');
 
-		controller.handleError(error);
+		controller.handleError(error, addedDetails);
 
 		expect(trackerTrackError).toHaveBeenCalledWith({
+			context: {
+				controller: {
+					type: 'abstract',
+					id: searchConfig.id,
+				},
+			},
+			details: JSON.stringify(addedDetails),
```

Comment:
Not sure we have to stringify here (if we did, we'd need to also do the `context` above)... we should send some test errors to find out.